src/arithmetic/arithmetic_expression.c

```
@@ -64,7 +64,11 @@ inline bool AR_EXP_IsOperation(const AR_ExpNode *exp) {
 	return exp->type == AR_EXP_OP;
 }
 
-bool AR_EXP_IsAttribute(const AR_ExpNode *exp, char **attr) {
+bool AR_EXP_IsAttribute
+(
```

cppcheck

[error] Null pointer dereference

(nullPointer)